subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -431,6 +431,7 @@ int fota_download(const char *host, const char *file,
 	struct download_client_cfg config = {
 		.pdn_id = pdn_id,
 		.frag_size_override = fragment_size,
+		.family = family
```

Comment:
question for the `fota_download` codeowners:
what is the reason for this separate config struct?  why not use `dlc.config`?